emu/src/render/ppu.rs

```diff
@@ -81,22 +80,24 @@ impl PixelProcessUnit {
             5 => {
                 // 06000000-06009FFF for Frame 0
                 // 0600A000-06013FFF for Frame 1
-                let selected_frame = memory.lcd_registers.get_frame_select();
-
-                // Bitmap mode
-                for y in 0..GBC_LCD_HEIGHT {
-                    for x in 0..GBC_LCD_WIDTH {
-                        let index_color = (y * GBC_LCD_WIDTH + x) * 2
-                            + (selected_frame * GBC_LCD_HEIGHT * GBC_LCD_WIDTH);
-                        let color: Color = [
-                            memory.video_ram[index_color],
-                            memory.video_ram[index_color + 1],
-                        ]
-                        .into();
-
-                        gba_lcd.set_gbc_pixel(x, y, color);
-                    }
-                }
+                // let selected_frame = memory.lcd_registers.get_frame_select();
+                //
+                // // Bitmap mode
+                // for y in 0..GBC_LCD_HEIGHT {
+                //     for x in 0..GBC_LCD_WIDTH {
+                //         let index_color = (y * GBC_LCD_WIDTH + x) * 2
+                //             + (selected_frame * GBC_LCD_HEIGHT * GBC_LCD_WIDTH);
+                //         // let color: Color = [
+                //         //     memory.video_ram[index_color],
+                //         //     memory.video_ram[index_color + 1],
+                //         // ]
+                //         // .into();
+                //         // FIXME
+                //         let color = Color(0);
+                //
+                //         gba_lcd.set_gbc_pixel(x, y, color);
+                //     }
+                // }
```

Comment:
Why commented?

Reply:
Right now we are not using this mode, just for being sure of what we need time to time